src/status/chat.nim

```diff
@@ -125,8 +125,8 @@ proc updateContacts*(self: ChatModel, contacts: seq[Profile]) =
     self.contacts[c.id] = c
   self.events.emit("chatUpdate", ChatUpdateArgs(contacts: contacts))
 
-proc init*(self: ChatModel) =
-  let chatList = status_chat.loadChats()
+proc init*(self: ChatModel, pubKey: string) =
+  var chatList = status_chat.loadChats()
```

Comment:
Are those changes needed?

Reply:
Ah, good catch. Both changes are needed in #1551 
Could've moved them there. Do you want me to?

Reply:
Nah it's k